src/signals/incident/components/form/MapInput/index.js

```diff
@@ -2,17 +2,25 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import mapLocation from 'shared/services/map-location';
-import MapInteractive from 'components/MapInteractive';
+import MapEditor from 'components/MapEditor';
 
 import Header from '../Header';
+import { DEFAULT_MARKER_POSITION } from '../../../../../shared/services/configuration/map-options';
```

Comment:
Can you use `'shared/services/configuration/map-options'` here instead of a relative path?

Reply:
sure!